Coding/Java/src/org/jmol/adapters/readers/quantum/QCJSONReader.java

```diff
@@ -0,0 +1,377 @@
+package org.jmol.adapter.readers.quantum;
+
```

Comment:
Can we simply put this in a `Coding/Java/...` folder. The additional nesting too much I would think.